app/views/webchat/ChildBenefitView.scala.html

```diff
@@ -32,7 +32,7 @@ <h1 @if(!isIvrRedirect){class="header-margin-top"}>@messages("child.benefit.titl
     <div class="govuk-inset-text">
         <p>@messages("global.opening.times.title")</p>
         <p>@messages("global.opening.times.8to7_30")</p>
-        <p>@messages("global.opening.times.saturday.8to4")</p>
+        <p>@messages("global.opening.times.saturday.8to330")</p>
```

Comment:
for consistency across the messages file can put _ for 3:30 PM
"global.opening.times.saturday.8to3_30"